### PR TITLE
BL-2316 Fix MRU List

### PR DESCRIPTION
**Pull request by gmartin7**
* case of extension was wrong (s/b bloomCollection)
* the change to control.Tag near the end of the method
   avoids a "Possible unintended reference comparison"
   compiler message

### CODE REVIEW

File: src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ private void OnLoad(object sender, EventArgs e)
 
 			foreach (Control control in tableLayoutPanel2.Controls)
 			{
-				if (control.Tag == "sendreceive")
+				if (control.Tag != null && control.Tag.ToString() == "sendreceive")
 					control.Visible = Settings.Default.ShowSendReceive;
```
Comment (JohnThomson) [inline src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs]: What does this have to do with the duplicate projects problem?

Comment (gmartin7) [inline src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs]: That particular bit? Nothing. My compiler was giving the complaint I mentioned in the PR description above, so I fixed it.
No real change in logic, just safer and clearer.

Comment (JohnThomson) [inline src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs]: Sorry, missed that.
